src/datatypes/src/value.rs

```diff
@@ -599,7 +599,11 @@ impl TryFrom<ScalarValue> for Value {
             | ScalarValue::Time32Second(_)
             | ScalarValue::Time32Millisecond(_)
             | ScalarValue::Time64Microsecond(_)
-            | ScalarValue::Time64Nanosecond(_) => {
```

Comment:
Looks like we already support these value types. We can fix them in other PR.

Reply:
I haven't update this branch against the latest develop. It will be removed